docs_src/security/Ch-Configuring-Add-On-Services.md

```diff
@@ -1 +1,240 @@
-# Placeholder
\ No newline at end of file
+# Configuring Add-on Service
+
+In the current EdgeX security serivces, we set up and configure all security related properties and environments
+for the existing default serivces like `core-data`, `core-metadata`, `device-virtual`, and so on.
+
+The settings and service environment variables are pre-wired and ready to run in secure mode without any update
+or modification to the Docker-compose files.  However, there are some pre-built add-on services like `device-camera`,
+`device-modbus` and some of application services (eg. `app-http-export`) are not pre-wired by default
```

Comment:
not totally agree with the suggestion; app-http-export is not the only one not pre-wired.  @lenny-intel 
if we put this way it may be kind of misleading example

Reply:
`like` infers these are not the only ones just like  `device-camera`, `device-modbus`  are not the only device service.